guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -14,6 +14,7 @@ endif::[]
 ifdef::foreman-el,katello,orcharhino,satellite[]
 * xref:migrating-ansible-content_{context}[]
 endif::[]
+* xref:post_upgrade-updating-templates-parameters_{context}[]
```

Comment:
One line down, pls. The references should have the same order as the included modules.